### PR TITLE
Deprecate ingest command

### LINKED ISSUE #5873: Import: Update documentation
Update documentation according to new implementation

### PR DESCRIPTION
**Pull request by N-o-Z**
### Linked Issue

Closes #5873

---

To be merged after https://github.com/treeverse/lakeFS/pull/5840

### CODE REVIEW

File: cmd/lakectl/cmd/ingest.go [hunk 1]
```diff
@@ -37,8 +37,9 @@ func stageWorker(ctx context.Context, client api.ClientWithResponsesInterface, w
 }
 
 var ingestCmd = &cobra.Command{
-	Use:   "ingest --from <object store URI> --to <lakeFS path URI> [--dry-run]",
-	Short: "Ingest objects from an external source into a lakeFS branch (without actually copying them)",
+	Deprecated: "use import command instead",
+	Use:        "ingest --from <object store URI> --to <lakeFS path URI> [--dry-run]",
```
Comment (idanovo) [inline cmd/lakectl/cmd/ingest.go:41]: Why do we explain how to use it if it's deprecated?

Comment (N-o-Z) [reply cmd/lakectl/cmd/ingest.go:41]: @idanovo thanks - though it's deprecated, it is still usable. Do you think we should remove it?

Comment (idanovo) [reply cmd/lakectl/cmd/ingest.go:41]: Ha, I thought it's not usable anymore

**Review by idanovo — APPROVED** [review]
LGTM

**Review by nopcoder — APPROVED** [review]

**Pull request merged**